### DISCUSSION TITLE
Moving aimmo-game-creator's game manager into the django codebase

### DISCUSSION THREAD

**Discussion by mrniket** [ocadotechnology/aimmo#1561] [Ideas]
Since the django project has access to the kubernetes cluster where the games run, there is no need to manually sync the state of the games in django with the state of the game on the cluster. We could have django as the single source of truth. This could potentially remove a lot of code which would be nice!

**Comment by dionizh** [comment]
@mrniket Yes this would be very useful! It'll tidy things up and speed up aimmo. Feel free to create an issue and 'claim' it if you'd like to work on it. Thank you! :)
